roles/repositories/README.md

```diff
@@ -0,0 +1,115 @@
+theforeman.foreman.repositories
+===============================
+
+This role defines Products and enabled Red Hat Repositories and Custom Repositories.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+- `products`: List of products to manage.
+  Each product is represented as a dictionary and can include `repository_sets` which represent Red Hat Repositories and should be used when the product name matches an existing Red Hat Product.
+  Each element of `repository_sets` must have a `name` and should specify the `basearch` and/or `releasever` only when multiple versions are available for that Product.
+  All repository sets for a Red Hat Product can be enabled by omitting `repository_sets` and instead specifying that the Product has `all_repositories: True`. When using this option it is also necessary to specify a list of repository `label`s for the Product (e.g. rhel-7-server-rpms). Be wary that this option can result in enabling a large number of unused repositories that, if added to sync plans, can greatly increase sync times and rapidly fill disk space.
```

Comment:
Why the `label` limitation? Can't it be enabled with `product` and `name` too?

Reply:
Ah, I see in https://github.com/theforeman/foreman-ansible-modules/pull/1027/files#r541297185

Okay for now I guess.